client/app/components/packages/pas-form/proposed-land-use-actions.hbs

```diff
@@ -8,7 +8,7 @@
       Note: This is preliminary information and may change following discussion with NYC Planning.
     </p>
     <div data-test-section="land-use-actions">
-      <LandUseAction
+      <Packages::LandUseAction
```

Comment:
Depending on future packages/whether this component is reusable, could eventually move to `<Packages::PasForm::LandUseAction>`